### PR TITLE
Fix part of #2057: Remove unnecessary RunOn from QuestionPlayerActivityTest

### PR DESCRIPTION
**Pull request by anandwana001**
## Explanation

Fix part of #2057 
Remove unnecessary RunOn from QuestionPlayerActivityTest

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

**Comment by rt4914** [comment]
@anandwana001 Assigning @BenHenning to this PR considering that though it works correctly but in that case why was it introduced in the first place.

**Comment by BenHenning** [comment]
We can't remove the checks in this case. The tests are *incorrectly* passing. The tick will always be visible because the text will never cause it to trigger the original bug. I have a branch sitting somewhere where I was starting to fix the underlying issue & I introduced new tests that explicitly fail without the fix in place which is a bit better than false positives. Nevertheless, could you please update the documentation in the tests to indicate that these incorrectly pass on Robolectric and shouldn't be re-enabled until the underlying issue is fixed @anandwana001?

**Comment by anandwana001** [comment]
> We can't remove the checks in this case. The tests are _incorrectly_ passing. The tick will always be visible because the text will never cause it to trigger the original bug. I have a branch sitting somewhere where I was starting to fix the underlying issue & I introduced new tests that explicitly fail without the fix in place which is a bit better than false positives. Nevertheless, could you please update the documentation in the tests to indicate that these incorrectly pass on Robolectric and shouldn't be re-enabled until the underlying issue is fixed @anandwana001?

As per the docs in `OppiaTestAnotations`, 
```
 * Note that this annotation only works if the test also has an [OppiaTestRule] hooked up.
```
But, this file, `QuestionPlayerActivityTest`, doesn't have the `OppiaTestRule` then who it is working? Although the tests are passing without it.

File: app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt [hunk 1]
```diff
@@ -210,7 +215,7 @@ class QuestionPlayerActivityTest {
   }
 
   // TODO(#2057): Remove when TextViews are properly measured in Robolectric.
-  @RunOn(TestPlatform.ESPRESSO)
+  @RunOn(TestPlatform.ESPRESSO) // Incorrectly pass on Robolectric and shouldn't be re-enabled
```
Comment (BenHenning) [inline app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt]: Typo: ``Incorrectly passes on Robolectric...``

Ditto below.

**Review by BenHenning — APPROVED** [review]
Thanks for clarifying the code & fixing the missing rule! @anandwana001 you're right that this was wrong before--the rule was missing. Good catch.

File: app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt (hunk 1 above)
Comment (anandwana001) [reply app/src/sharedTest/java/org/oppia/android/app/topic/questionplayer/QuestionPlayerActivityTest.kt]: Ah, looks like I need Grammarly in Android Studio. 
Done

**Review by BenHenning — APPROVED** [review]
Thanks!

**Pull request merged**